Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM python:2.7.8-wheezy
```

Comment:
Change base image to `python:2`, so the latest version of the 2.x series is always used.